index.html

```diff
@@ -0,0 +1,206 @@
+            <li>javascript</li>
+          </ul>
+          <button class="preview_button outlinings">See Project</button>
+        </article>
+        <article class="card flex white-back">
+          <img src="./images/project_3.png" alt="tonic project preview" />
+          <h3>Tonic</h3>
+          <ul class="flex project_details">
+            <li>CANOPY</li>
+            <li>BACKEND DEV</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex project_coding_langs">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="preview_button outlinings">See Project</button>
+        </article>
+        <article class="card flex white-back">
+          <img src="./images/project_4.png" alt="Multi-Post project preview" />
+          <h3>Multi-Post Stories</h3>
+          <ul class="flex project_details">
+            <li>CANOPY</li>
+            <li>BACKEND DEV</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex project_coding_langs">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="preview_button outlinings">See Project</button>
+        </article>
+      </section>
+      <section class="about white-back">
+        <div>
+          <h1>
+            About <br />
+            Myself
+          </h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div class="socials">
+            <h6>LET'S CONNECT</h6>
+            <ul class="icons_social__container flex">
+              <li>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-instagram"></i></a>
+              </li>
+            </ul>
+          </div>
+          <button class="outlinings">Get My Resume</button>
+          <ul class="about__stacks">
+            <li class="about_stacks__item" style="border-bottom: 0px">
+              <span>Languages</span>
+              <i class="fa fa-angle-down"></i>
```

Comment:
As per the Figma design, the color of these icons are `#ACB7C3`. The default color you are using here seems to be darker. Kindly check it out again.